### PR TITLE
Node Feedback

### PR DESCRIPTION
**Pull request by benjaminstrasser**
Collection of Feedback regarding node SDK
Signed-off-by: Benjamin Strasser <bp.strasser@gmail.com>

### CODE REVIEW

**Comment by gerhard** [comment]
This looks good to me - love the small PR!

You may want to rebase on top of `main` @benjaminstrasser to get this improvement https://github.com/dagger/dagger/pull/3875

Leaving it to you @benjaminstrasser to convert this to **Ready for review**.

Trusting @slumbering @vikram-dagger @dolanor to approve & merge.

👍

File: examples/nodejs/package.json [hunk 1]
```diff
@@ -6,8 +6,8 @@
   "repository": "https://github.com/dagger/dagger",
   "author": "Jean-François Farge",
   "license": "MIT",
-  "dependencies": {
-    "@dagger.io/nodejs-sdk": "https://github.com/slumbering/dagger-nodejs-sdk.git",
+  "devDependencies": {
+    "@dagger.io/dagger": "^0.1.0-alpha.4",
```
Comment (slumbering) [inline examples/nodejs/package.json:10]: alpha packages are going to be removed. 
`"@dagger.io/dagger": "^0.1.0",` should be the published package version.
thanks

**Review by slumbering — CHANGES_REQUESTED** [review]
Many thanks for your help :)

**Review by slumbering — APPROVED** [review]
LGTM 🚀

**Pull request merged**